### ISSUE TITLE
Backwards discretionary ligature of yod and vav

### ISSUE THREAD

**Issue by dscorbett** [googlei18n/noto-fonts#1229]
### Font
NotoSansHebrew-Regular.ttf
NotoSerifHebrew-Regular.ttf

### Where the font came from, and when
Site: https://noto-website-2.storage.googleapis.com/pkgs/NotoSansHebrew-unhinted.zip
Site: https://noto-website-2.storage.googleapis.com/pkgs/NotoSerifHebrew-unhinted.zip
Date: 2018-03-04

### Font version
Version 2.000;GOOG;noto-source:20170915:90ef993387c0

### Issue
'dlig' applied to yod and vav produces a sequence of vav and yod.

### Character data
יו
U+05D9 HEBREW LETTER YOD
U+05D5 HEBREW LETTER VAV

### Screenshot
![יו](https://user-images.githubusercontent.com/1124347/41663006-6bdf0716-7470-11e8-87ac-8c851a156ff3.png)![יו](https://user-images.githubusercontent.com/1124347/41663008-6bfe3e74-7470-11e8-93a3-31dff9ccbae8.png)

**Comment by dscorbett** [comment]
This bug is also present in Noto Rashi Hebrew.
![יו](https://user-images.githubusercontent.com/1124347/71424925-31f65100-2665-11ea-8967-f9bdac69e3b7.png)

**Comment by nizarsq** [comment]
This issue seem to be fixed. @dscorbett any opinion?
<img width="926" alt="Screen Shot 2020-08-04 at 3 18 10 PM" src="https://user-images.githubusercontent.com/22158882/89350891-b5e5d200-d665-11ea-80ee-c0145bcaa1ec.png">

**Comment by dscorbett** [comment]
Did you enable 'dlig' for that test?

**Comment by nizarsq** [comment]
Yes tested when dlig is enabled and disabled. Attached screenshot is for enabled dlig.
<img width="1387" alt="Screen Shot 2020-08-04 at 4 37 02 PM" src="https://user-images.githubusercontent.com/22158882/89355641-bc2d7b80-d670-11ea-8dec-3eb35bee647c.png">

**Comment by dscorbett** [comment]
That test does not enable 'dlig' correctly. The CSS declaration is `font-font-feature-settings: dlig` but it should be `font-feature-settings: 'dlig'`.

**Comment by nizarsq** [comment]
> That test does not enable 'dlig' correctly. The CSS declaration is `font-font-feature-settings: dlig` but it should be `font-feature-settings: 'dlig'`.

Thanks, I already updated the code.

**Comment by nizarsq** [comment]
<img width="1313" alt="Screen Shot 2020-08-04 at 7 05 18 PM" src="https://user-images.githubusercontent.com/22158882/89363593-729b5b80-d685-11ea-8e13-e89d92ccebbe.png">
<img width="829" alt="Screen Shot 2020-08-04 at 7 06 00 PM" src="https://user-images.githubusercontent.com/22158882/89363636-8a72df80-d685-11ea-837e-230f61bc95e4.png">

**Comment by dscorbett** [comment]
The CSS declaration is still wrong. It should be `font-feature-settings: 'dlig'`.

**Comment by nizarsq** [comment]
> The CSS declaration is still wrong. It should be `font-feature-settings: 'dlig'`.

My bad. I didn't see that I typed in "font-" twice. And copied for each font. I'll update the bug with correct version.

**Comment by nizarsq** [comment]
<img width="823" alt="Screen Shot 2020-08-04 at 9 25 24 PM" src="https://user-images.githubusercontent.com/22158882/89371728-04609400-d699-11ea-803d-c0d39351a16e.png">

**Comment by kosmynkab** [comment]
The order in the feature code was wrong - it was automatically generated for the vavyod-hb ligature as sub yod-hb vav-hb instead of other way around. Resolved across all Noto Sans, Serif and Rashi versions (will be pushed in the collective update of multiple issue resolution).
